docs/guides/auth.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Authentication
+description: "Learn how you can provide granular access to types and fields in GraphQL schema with the help of authentication providers"
+---
+
+## What is Authentication?
+
+Authentication in GraphQL refers to the process of verifying the identity of a user before allowing access to specific fields in GraphQL schema. Essentially, it's about controlling who can see or interact with certain pieces of data exposed by GraphQL API. This process is crucial for following reasons:
+
+- **Protect Sensitive Information** Some fields may contain sensitive information that should not be accessible to unauthenticated users.
+- **Prevent Unauthorized Access** Ensuring that authenticated users can query certain fields helps prevent unauthorized access and potential data breaches.
+- **Customized User Experiences** Authentication allows for personalized queries based on the user's identity.
+
+The authentication could be implemented by different credential validation mechanisms, like:
```

Comment:
Expand a bit on how it's different from existing auth mechanisms particularly around field level auth.